<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageModeEtPompe</class>
 <widget class="QMainWindow" name="PageModeEtPompe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_semiauto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SEMI-AUTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_auto">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AUTOMATIQUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_continuer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONTINUER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
